data/starts.txt

```diff
@@ -96,7 +97,7 @@ start "hai space after FW and HR"
 
 conversation "hai intro"
 	scene "scene/hai shipyard"
-	`	You've never been to the shipyard before, and as you arrive, it already feels like you're traveling to another world. The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
+	`	You've never been to the shipyard before, and going to the poles already feels like you're traveling to another world. The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
```

Comment:
Honestly I prefer the previous version. Also, it feels weird since nobody said a word about the shipyard being near the poles.

Reply:
![image](https://user-images.githubusercontent.com/94366726/214269370-e0a16a4d-c820-418b-8357-240140feee00.png)

Reply:
FL in _this_ start has never read that description.

Reply:
The player doesn't lose information doing this new start, though...? It's taking information that's true about the world and extrapolating an interesting concept from it. I don't understand where this part of the comment is coming from.

Reply:
The phrase, "going to the poles", would not make much sense to a player. Could you please choose "North Pole" or "South Pole"?